content/docs/releases/README.md

```diff
@@ -16,18 +16,18 @@ and other world events.
 <a id="supported-releases"></a>
 ## Currently supported releases
 
-|   Release    | Release Date |      End of Life       | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
+| Release      | Release Date | End of Life            | [Supported Kubernetes versions][s] | [Supported OpenShift versions][s] |
 |--------------|:------------:|:----------------------:|:----------------------------------:|:---------------------------------:|
-|     [1.13][] | Sep 12, 2023 |    Release of 1.15     |            1.23 → 1.28             |           4.10 → 4.15             |
-| [1.12 LTS][] | May 19, 2023 |      May 19, 2025      |            1.22 → 1.28             |            4.9 → 4.15             |
+| [1.14][]     | Feb 3, 2024  | ~4 months post release | 1.24 → 1.29                        | 4.11 → 4.15                       |
```

Comment:
nitpick (non-blocking): I'd maybe say "release of 1.16" rather than adding in another date - but it doesn't really matter!